scripts/version-repo.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+REPO_VER=v`cat ../CHANGELOG.md | grep "##" | sort -r | head -n 1 | cut -d" " -f2`
```

Comment:
I don't think this will sort semver correctly.

```
cat test.txt | sort -r
3.9.4
1.9.3
0.9.11
0.3.0
0.11.2
0.1.0
```

Reply:
In this case wouldn't 3.9.4 be the desired version?